sdk/simulation/uinst/u_instructions.cpp

```diff
@@ -223,6 +225,12 @@ void sig_handler_sim(int signum, siginfo_t *siginfo, void *priv)
                             p_ssa_gpr->exit_info.valid = 0;
                         }
                         tcs->cssa +=1;
+                    } else {
+                        if (signum == SIGRT_INTERRUPT) {
+                            // will call urts sig_handler and return
+                            tcs_target_state = TCS_STATE_ACTIVE;
```

Comment:
I think the set status should be moved to line 238.

Reply:
Done. Thanks for your review.